### PR TITLE
Version 2: Add Certificate download button

### PR DESCRIPTION
**Pull request by OleksandrSPV**
image

### CODE REVIEW

File: src/components/certificates-list/CertificatesList.tsx [hunk 1]
```diff
@@ -63,7 +66,8 @@ export const CertificatesList: React.FunctionComponent<
         </TableHeader>
         <TableBody>
           {certificates.map((certificate) => {
-            const { id, serialNumber, type, label, notAfter } = certificate;
+            const { id, serialNumber, type, label, notAfter, raw } =
```
[the remark this answers is not in the document]
Comment (OleksandrSPV) [reply src/components/certificates-list/CertificatesList.tsx:69]: image
There was many changes after I added this rule, can we add it in separate PR later? And there is another conflict with Prettier, I need found the solution.

File: src/utils/download-certificate.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const url = window.URL.createObjectURL(blob);
+  const a = document.createElement("a");
+
+  a.href = url;
+  a.download = name;
+  a.click();
+  window.URL.revokeObjectURL(url);
+}
+
+export function downloadCertificate(
```
[the remark this answers is not in the document]
Comment (OleksandrSPV) [reply src/utils/download-certificate.ts:14]: Ok, shall we use this temporary solution and then after api changes I replace it.

[the remark this answers is not in the document]
Comment (donskov) [reply src/utils/download-certificate.ts:14]: Try to use the new version of `@peculiar/certificates-viewer`. It should includes download utils.

File: src/components/certificates-list/CertificatesList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (donskov) [reply src/components/certificates-list/CertificatesList.tsx:69]: Ok. Let's create separate PR.

File: src/utils/download-certificate.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (OleksandrSPV) [reply src/utils/download-certificate.ts]: Done @donskov

**Review by donskov — APPROVED** [review]

**Pull request merged**